test/test_manifest.json

```diff
@@ -12140,5 +12140,13 @@
     "type": "eq",
     "forms": true,
     "link": true
+  },
+  {
+    "id": "bug1974436",
+    "file": "pdfs/bug1974436.pdf",
```

Comment:
The file is not actually includes in this PR (and `.gitignore` is not updated accordingly) which causes the bots to fail.